<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pushBackElement</class>
 <widget class="QDialog" name="pushBackElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>283</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="fioLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Должность:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="dolzhnostSpinBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QSpinBox" name="otdelSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ф.И.О:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Номер отдела:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="dayComboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>29</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>31</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="monthComboBox">
        <item>
         <property name="text">
          <string>Январь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Февраль</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Март</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Апрель</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Май</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Июнь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Июль</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Август</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сентябрь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Октябрь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ноябрь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Декабрь</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="yearSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Дата начала работы:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>окей</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
